deploy/018-native-token-gateway.ts

```diff
@@ -99,6 +106,8 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
       args: [vWNativeInfo.address],
       log: true,
       autoMine: true,
+      skipIfAlreadyDeployed: true,
+      // maxFeePerGas: "200000000"  // Needed for zksync
```

Comment:
Remove it or apply the dynamic solution we have applied in other scenarios

Reply:
[310033a](https://github.com/VenusProtocol/isolated-pools/pull/425/commits/310033ab2e08ddde1bc2bd8f807b76f1cf0e57ed)